.changeset/short-frogs-drop.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-catalog-graphql': patch
+---
+
+Upgrade @graphql-codegen/cli to latest
```

Comment:
Wrap with `` ` `` to make it an inline code block, which avoids the vocab update.

Reply:
Nice one. Thanks for the tip.